.github/workflows/terraform.yaml

```diff
@@ -0,0 +1,78 @@
+name: "Terraform"
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Is terraform racist as well and will accept master in place of main?

Reply:
Fixed